services/community-site/src/utils/compareDelegationTo.ts

```diff
@@ -0,0 +1,12 @@
+import { ethers } from 'ethers';
+
+export const compareDelegationTo = (delegation: string, to = '1000'): boolean => {

```

Comment:
Can we take this one abstraction level above and simply use it to compare a BigNumber to a float/Number ? This way we can use it to compare `event.target.value` in every sencario in both components

Reply:
Hmm I thought of doing something similar but I don't think we can use it in multiple places except these two where I am using it now. Because we compare it either with float or BigNumber, either lower or greater. So it can`t be that reusable. Also it is a headache to test each scenarion.